File: scala/org/softlang/company/Person.scala
package org.softlang.company

/**
 * A person has a name and an address.
 */
class Person {
  private var name: String = ""
  private var address: String = ""

  def getName = name
  def setName(name: String) = {
    this.name = name
  }

  def getAddress = address
  def setAddress(address: String) = {
    this.address = address
  }

}

File: aspectJ/org/softlang/company/Person.java
package org.softlang.company;

/**
 * A person has a name and an address
 * 
 */
public class Person {

	private String name;
	private String address;

	public Person() {
		name = "";
		address = "";
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

}
